content/W2017/W17-event-dinner-with-professors.md

```diff
@@ -0,0 +1,20 @@
+title: Dinner with the Professors
+Date: 2017-03-1
+Category: Events
+Tags: events, social
+Slug: dinner-with-professors-W17
+Author: Talia McCormick
+Summary: Student-faculty dinner.
+
+Come have dinner with the brilliant women who are professors at Waterloo! This is a wonderful opportunity to learn about their research, academia, teaching, and your own academic interests. 
+
+Note: This event is women-only and requires registration.
+
+## Event Details ##
+
++ **Who:** [Cis & trans women and non-binary individuals]({filename}/pages/faq.md)
++ **What:** Student-faculty dinner
++ **Where:** University Club
++ **When:** March 8, 2017 6:30PM
++ **Note:** Tickets for students are sold for $10 in MathSoc, until Wednesday, March 1. Sign-up for professors and for grad students who are interested in attending as mentors is free and run through eventbrite (here: goo.gl/B8PQHL ). Registration is required and is first-come first-serve. If you can not make it to the MathSoc office in-person, please email us at wics-ugrad@lists.uwaterloo.ca to find a workaround.
```

Comment:
This note should go in the main part of the post. The lines also need to be 80 char long. 

The link to the eventbrite should be embedded. i.e [evenbrite](goo.gl/B8PQHL).